src/transformers/utils/import_utils.py

```diff
@@ -39,16 +39,32 @@
 
 # TODO: This doesn't work for all packages (`bs4`, `faiss`, etc.) Talk to Sylvain to see how to do with it better.
 def _is_package_available(pkg_name: str, return_version: bool = False) -> Union[Tuple[bool, str], bool]:
-    # Check we're not importing a "pkg_name" directory somewhere but the actual library by trying to grab the version
+    # Check if the package spec exists to avoid importing a local directory
```

Comment:
```suggestion
    # Check if the package spec exists and grab its version to avoid importing a local directory
```

Reply:
The PR is editable so no worry if you want to do this minor edits on your side.

Reply:
you can click `commit suggestion`, but I can also click it 😄